### ISSUE TITLE
{{cdn}} injection helper with `webdav:` is not using local `/assets` directory

### ISSUE THREAD

**Issue by brandonjp** [bigcommerce/cornerstone#1593]
Not sure if this should be with Stencil or Cornerstone, please close the incorrect post

### Expected behavior
`{{cdn 'webdav:img/img.jpg'}}` should transform locally to: `/assets/cdn/img/img.jpg`

### Actual behavior
`{{cdn 'webdav:img/img.jpg'}}` is transformed locally to: `/content/img/img.jpg`

For me, this:
![PixelSnap 2019-11-06 at 20 44 36@2x](https://user-images.githubusercontent.com/199599/68356782-daae1880-00d8-11ea-9a5b-ff3566e6705d.png)

Is transformed to this:
![PixelSnap 2019-11-06 at 20 45 35@2x](https://user-images.githubusercontent.com/199599/68356842-08935d00-00d9-11ea-8533-d7e45df1c7fc.png)

### Notes
Based on docs (see below), it's my understanding that using `{{cdn 'webdav:img/img.jpg'}}` locally should produce a url pointing to `/assets/cdn/img/img.jpg` (and should not require additional configuration in `config.json` or elsewhere, as is required for the "CDN Custom Endpoints" mentioned in the docs). However, this doesn't seem to be the case. The URL produced seems to be the production url `/content/img/img.jpg`.

I've tried my files in a variety of locations. Here is my folder structure:
![PixelSnap 2019-11-06 at 21 29 36@2x](https://user-images.githubusercontent.com/199599/68358185-6033c780-00dd-11ea-936a-b41c7e381a42.png)

Alternatively, defining a custom CDN endpoint does seem to render as expected. In this example, I've defined `localdav` and am using `{{cdn 'localdav:img/img.jpg'}}` in the template.
![PixelSnap 2019-11-06 at 21 32 37@2x](https://user-images.githubusercontent.com/199599/68358196-69249900-00dd-11ea-8f91-29c1adf0b526.png)

From https://developer.bigcommerce.com/stencil-docs/reference-docs/handlebars-helpers-reference
![PixelSnap 2019-11-06 at 20 47 24@2x](https://user-images.githubusercontent.com/199599/68356666-8440da00-00d8-11ea-84dc-639645c1b36b.png)

From https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/checking-a-themes-size
![PixelSnap 2019-11-06 at 21 23 17@2x](https://user-images.githubusercontent.com/199599/68357775-0252b000-00dc-11ea-8e48-36afb79dc000.png)

**Comment by bookernath** [comment]
The `webdav:` syntax will pull assets from the object storage hosted on BigCommerce ([accessed via WebDav](https://support.bigcommerce.com/s/article/File-Access-WebDAV)), not from your local directories.

To simply reference files from your assets directly, simply remove the `webdav:` prefix.

For example:

https://github.com/bigcommerce/cornerstone/blob/4e67c81ba03189123e943bf2e54059ebb86499d3/templates/layout/base.html#L23

https://github.com/bigcommerce/cornerstone/blob/4e67c81ba03189123e943bf2e54059ebb86499d3/templates/components/account/payment-methods-list.html#L20

**Comment by brandonjp** [comment]
Right, I agree that's _how_ it works. But the docs imply that locally it should pull from the local `/assets/cdn`. Plus, the fact that you can define a custom cdn seems to imply such, as well.  Otherwise, the use of the `{{cdn}}` helper seems, well, less useful. 

From the docs, [Checking a Theme's Size > Organizing Your Theme](https://developer.bigcommerce.com/stencil-docs/deploying-a-theme/checking-a-themes-size) :
> To stage your theme for CDN delivery, you would locally store your large static assets within your theme’s assets/cdn/ subdirectory. ... Then, as you develop your theme, reference each of these assets using Stencil’s cdn custom Handlebars helper, like below: `<img src="{{cdn "webdav:img/image.jpg"}}">`
